roles/ansible/tower/config-ansibletower/tasks/install.yml

```diff
@@ -56,3 +56,12 @@
 
 - name: "Download and Install the 'oc' client for OpenShift interactions"
   shell: curl {{ oc_client_download_url }} | tar -C /bin/ -xzf -
+
+- name: "Copy certs"
+  copy:
+    src: "{{ item }}"
+    dest: /etc/tower/
```

Comment:
The destination files will always have to be named `tower.cert` and `tower.key`, but the source filename may not always be, so please update to enforce the naming at the destination while still allowing any name at source. 

see bullet item 2 in the installation note of Ansible Tower:
https://docs.ansible.com/ansible-tower/latest/html/installandreference/install_notes_reqs.html#installation-notes